submissions/damilolaSalami/html-form/signUp.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="signUp.css">
+    <title>Buy Me</title>
+</head>
+<body>
+    <nav>
+        <div class="header">
+            <div class="logo">
+                <img src="images/logo.jpg" alt="logo">
+            </div>
+           
+            <div class="headerlinks">
+                <ul>
+                    <li><a href="#">Product page</a></li>
+                    <li><a href="#">Contact Us</a></li>
+                    <li><a href="siguppage.html">Sign up</a></li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    
+<div class="mainbody">
+    <div class="registration container">
+        <h1>Register your<br>Account</h1> <br>
+        <p>Registering an account gives you <br>
+           access to sweet promo and events <br>
+           available to only registered <br>
+           customers.</p>
+        <br>
+        <p>Already got an account? <a href="#">Sign in</a></p> 
+    </div>
+
+    <section>
```

Comment:
Remove all `br` elements from this file and style the input field appropriately with css. @queensaldam

Reply:
Remove all br elements from this file and style the input field appropriately with css. @queensaldam